_schemaV2.graphql

```diff
@@ -1023,6 +1023,9 @@ type Artwork implements Node & Searchable & Sellable {
   # Is this artwork part of an auction that is currently running?
   isBiddable: Boolean
 
+  # Is this artwork part of an auction that is currently running?
+  canRequestLotConditionsReport: Boolean
```

Comment:
I'd be curious to hear about this name. The name `canRequestLotConditionsReport` is bit of a convention-beaking name as we have many more `isAdjective` fields, but it reads better from a front-end point of view. Any suggestions or feedback here?